### PR TITLE
Integrate Metabox in the Github workflow

### PR DESCRIPTION
**Pull request by pieqq**
## Description

Create a GitHub action to run Metabox on a variety of configs (Checkbox local, remote, Ubuntu 18.04/20.04/22.04) when a PR is approved.

## Resolved issues

CHECKBOX-451

### CODE REVIEW

**Review by yphus — CHANGES_REQUESTED** [review]
with `runs-on: ubuntu-20.04` you can have a config for 16.04 (no cgroups issues with a host running 20.04)

**Comment by pieqq** [comment]
I've made the changes required. Hopefully we can run everything!

**Comment by pieqq** [comment]
I want to amend this PR to prevent Metabox from being run if some non-Checkbox changes are approved

**Comment by yphus** [comment]
> I want to amend this PR to prevent Metabox from being run if some non-Checkbox changes are approved

I've not find a way to combine both `pull_request_review` events and path filters constraints.

**Comment by pieqq** [comment]
Indeed, it looks like the paths filter is only available for [`pull_request` and `pull_request_target`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore). Annoying!

Let's land it this way, then, we'll see how long it is to run.

File: .github/workflows/metabox.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  # Allow manual trigger
+  workflow_dispatch:
+
+jobs:
+  Metabox:
+    if: github.event.review.state == 'approved'
+    strategy:
+      matrix:
+        # Version of Ubuntu
+        os: [18.04, 20.04, 22.04]
```
Comment (yphus) [inline .github/workflows/metabox.yaml:15]: Please add 16.04 here too

**Review by yphus — CHANGES_REQUESTED** [review]
Missing 16.04 in the os matrix

**Review by yphus — APPROVED** [review]
LGTM

**Review by yphus — APPROVED** [review]
+1

**Review by yphus — APPROVED** [review]
+1 (after pip install --upgrade)

**Review by yphus — APPROVED** [review]
+1

**Review by yphus — APPROVED** [review]
+1

**Pull request merged**